src/pages/task/taskTabs/testsTable/TestsTableCore.tsx

```diff
@@ -190,18 +195,47 @@ const columnsTemplate: ColumnProps<TestResult>[] = [
   },
   {
     title: <span data-cy="logs-column">Logs</span>,
-    width: 150,
+    width: 230,
     dataIndex: "logs",
     key: "logs",
     sorter: false,
     render: ({
+      execution,
+      groupID,
       htmlDisplayURL,
+      id,
       rawDisplayURL,
+      taskId,
+      lineNum,
     }: {
+      execution: number;
+      groupID: string;
       htmlDisplayURL: string;
+      id: string;
       rawDisplayURL: string;
+      taskId: string;
+      lineNum: number;
     }): JSX.Element => (
       <>
+        {!isLobsterLink(htmlDisplayURL) && (
```

Comment:
Just trying to understand whats going on.
if `htmlDisplayURL` **IS** a lobster link then we do **NOT** show the lobster button and only show the `HTML` button

if `htmlDisplayURL` **IS NOT** a lobster link then we **DO** show the lobster button and generate a lobster link?


This logic is a little confusing why not just show the lobster button if a link is a lobster link?

Reply:
mongodb-mongo-* projects already link to logkeeper from the "HTML" button in the test table ([example](https://spruce.mongodb.com/task/mongodb_mongo_master_enterprise_rhel_80_64_bit_dynamic_required_noPassthrough_1_enterprise_rhel_80_64_bit_dynamic_required_patch_e9f91c18323784288cd89cbbb70bc0eca8e89aae_60302dc032f4173ac4381c35_21_02_19_21_29_50/tests?execution=0)).
in this case we don't want to render a "Lobster" button

Reply:
I think the logic still applies if we are linking to lobster shouldn't we indicate that? Regardless of whether or not spruce generates the link or its the default link. How do you feel about that?

Reply:
good idea